<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>944</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="employe">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>901</width>
      <height>571</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(21, 139, 151);
</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="ajout">
     <attribute name="title">
      <string/>
     </attribute>
     <widget class="QPushButton" name="bin_image">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>490</y>
        <width>151</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #b1b1b1;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);

 

    border-width: 3px;

    border-color: #ffffff;

    border-style: solid;

    border-radius: 10;



    padding: 3px;

    font-size: 12px;

    padding-left: 5px;

    padding-right: 5px;</string>
      </property>
      <property name="text">
       <string>inserer une image </string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../plus.png</normaloff>../plus.png</iconset>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_image_2">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>130</y>
        <width>241</width>
        <height>281</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>421</width>
        <height>491</height>
       </rect>
      </property>
      <property name="title">
       <string>Ajout d'un employé </string>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>80</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Nom&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_nom">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>70</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #b1b1b1;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);

 

    border-width: 3px;

    border-color: #ffffff;

    border-style: solid;

    border-radius: 10;



    padding: 3px;

    font-size: 12px;

    padding-left: 5px;

    padding-right: 5px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Prenom&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_prenom">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>130</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #b1b1b1;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);

 

    border-width: 3px;

    border-color: #ffffff;

    border-style: solid;

    border-radius: 10;



    padding: 3px;

    font-size: 12px;

    padding-left: 5px;

    padding-right: 5px;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>190</y>
         <width>161</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Date de naissance&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_date_naissance">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>190</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #b1b1b1;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);

 

    border-width: 3px;

    border-color: #ffffff;

    border-style: solid;

    border-radius: 10;



    padding: 3px;

    font-size: 12px;

    padding-left: 5px;

    padding-right: 5px;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>250</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Numéro&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_numero_telephone">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>250</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #b1b1b1;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);

 

    border-width: 3px;

    border-color: #ffffff;

    border-style: solid;

    border-radius: 10;



    padding: 3px;

    font-size: 12px;

    padding-left: 5px;

    padding-right: 5px;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>310</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Salaire&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_salaire">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>310</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #b1b1b1;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);

 

    border-width: 3px;

    border-color: #ffffff;

    border-style: solid;

    border-radius: 10;



    padding: 3px;

    font-size: 12px;

    padding-left: 5px;

    padding-right: 5px;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>370</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;ID&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_id">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>370</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #b1b1b1;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);

 

    border-width: 3px;

    border-color: #ffffff;

    border-style: solid;

    border-radius: 10;



    padding: 3px;

    font-size: 12px;

    padding-left: 5px;

    padding-right: 5px;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_ajouter">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>450</y>
         <width>111</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #b1b1b1;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);

 

    border-width: 3px;

    border-color: #ffffff;

    border-style: solid;

    border-radius: 10;



    padding: 3px;

    font-size: 12px;

    padding-left: 5px;

    padding-right: 5px;</string>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../plus.png</normaloff>../plus.png</iconset>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="le_image">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>440</y>
        <width>211</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page2">
     <attribute name="title">
      <string/>
     </attribute>
     <widget class="QPushButton" name="pb_supprimer">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>390</y>
        <width>111</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #b1b1b1;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);

 

    border-width: 3px;

    border-color: #ffffff;

    border-style: solid;

    border-radius: 10;



    padding: 3px;

    font-size: 12px;

    padding-left: 5px;

    padding-right: 5px;</string>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../trash.png</normaloff>../trash.png</iconset>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_id_supp">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>390</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #b1b1b1;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);

 

    border-width: 3px;

    border-color: #ffffff;

    border-style: solid;

    border-radius: 10;



    padding: 3px;

    font-size: 12px;

    padding-left: 5px;

    padding-right: 5px;</string>
      </property>
     </widget>
     <widget class="QTableView" name="tab_employe">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>20</y>
        <width>391</width>
        <height>251</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="recherher">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>440</y>
        <width>111</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #b1b1b1;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);

 

    border-width: 3px;

    border-color: #ffffff;

    border-style: solid;

    border-radius: 10;



    padding: 3px;

    font-size: 12px;

    padding-left: 5px;

    padding-right: 5px;</string>
      </property>
      <property name="text">
       <string>rechercher </string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../256556278_4312998488811986_5619313483223777488_n.png</normaloff>../256556278_4312998488811986_5619313483223777488_n.png</iconset>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>340</y>
        <width>211</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Veuillez entrer l'ID de l'employé </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="id_chercher">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>440</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #b1b1b1;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);

 

    border-width: 3px;

    border-color: #ffffff;

    border-style: solid;

    border-radius: 10;



    padding: 3px;

    font-size: 12px;

    padding-left: 5px;

    padding-right: 5px;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="tri">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>390</y>
        <width>121</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #b1b1b1;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);

 

    border-width: 3px;

    border-color: #ffffff;

    border-style: solid;

    border-radius: 10;



    padding: 3px;

    font-size: 12px;

    padding-left: 5px;

    padding-right: 5px;
</string>
      </property>
      <property name="text">
       <string>trier </string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../291613227_370296138513623_6295941384685589861_n.png</normaloff>../291613227_370296138513623_6295941384685589861_n.png</iconset>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>431</width>
        <height>321</height>
       </rect>
      </property>
      <property name="title">
       <string>Modifier </string>
      </property>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>50</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="edit_id">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>50</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #b1b1b1;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);

 

    border-width: 3px;

    border-color: #ffffff;

    border-style: solid;

    border-radius: 10;



    padding: 3px;

    font-size: 12px;

    padding-left: 5px;

    padding-right: 5px;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>NOM</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="edit_nom">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>100</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #b1b1b1;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);

 

    border-width: 3px;

    border-color: #ffffff;

    border-style: solid;

    border-radius: 10;



    padding: 3px;

    font-size: 12px;

    padding-left: 5px;

    padding-right: 5px;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>PRENOM</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="edit_prenom">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>150</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #b1b1b1;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);

 

    border-width: 3px;

    border-color: #ffffff;

    border-style: solid;

    border-radius: 10;



    padding: 3px;

    font-size: 12px;

    padding-left: 5px;

    padding-right: 5px;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_19">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>60</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>DATE DE NAISSANCE</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="edit_date">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>50</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #b1b1b1;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);

 

    border-width: 3px;

    border-color: #ffffff;

    border-style: solid;

    border-radius: 10;



    padding: 3px;

    font-size: 12px;

    padding-left: 5px;

    padding-right: 5px;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>110</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>NUMERO TELEPHONE</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="edit_numero">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>100</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #b1b1b1;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);

 

    border-width: 3px;

    border-color: #ffffff;

    border-style: solid;

    border-radius: 10;



    padding: 3px;

    font-size: 12px;

    padding-left: 5px;

    padding-right: 5px;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>160</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>SALAIRE</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="edit_salaire">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>150</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #b1b1b1;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);

 

    border-width: 3px;

    border-color: #ffffff;

    border-style: solid;

    border-radius: 10;



    padding: 3px;

    font-size: 12px;

    padding-left: 5px;

    padding-right: 5px;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>270</y>
         <width>111</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #b1b1b1;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);

 

    border-width: 3px;

    border-color: #ffffff;

    border-style: solid;

    border-radius: 10;



    padding: 3px;

    font-size: 12px;

    padding-left: 5px;

    padding-right: 5px;</string>
       </property>
       <property name="text">
        <string>UPDATE</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../edit-button.png</normaloff>../edit-button.png</iconset>
       </property>
      </widget>
      <widget class="QLabel" name="edit_image">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>200</y>
         <width>141</width>
         <height>111</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #b1b1b1;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);

 

    border-width: 3px;

    border-color: #ffffff;

    border-style: solid;

    border-radius: 10;



    padding: 3px;

    font-size: 12px;

    padding-left: 5px;

    padding-right: 5px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>210</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Image</string>
       </property>
      </widget>
      <widget class="QPushButton" name="le_modd">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>240</y>
         <width>41</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #b1b1b1;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);

 

    border-width: 3px;

    border-color: #ffffff;

    border-style: solid;

    border-radius: 10;



    padding: 3px;

    font-size: 12px;

    padding-left: 5px;

    padding-right: 5px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../plus.png</normaloff>../plus.png</iconset>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="aff_image">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>340</y>
        <width>221</width>
        <height>171</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pdf">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>390</y>
        <width>111</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #b1b1b1;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);

 

    border-width: 3px;

    border-color: #ffffff;

    border-style: solid;

    border-radius: 10;



    padding: 3px;

    font-size: 12px;

    padding-left: 5px;

    padding-right: 5px;</string>
      </property>
      <property name="text">
       <string>PDF</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../280725885_2985139278402288_3140261754862112131_n.png</normaloff>../280725885_2985139278402288_3140261754862112131_n.png</iconset>
      </property>
     </widget>
     <widget class="QLineEdit" name="path">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>280</y>
        <width>341</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="le_fich_excel">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>440</y>
        <width>111</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #b1b1b1;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);

 

    border-width: 3px;

    border-color: #ffffff;

    border-style: solid;

    border-radius: 10;



    padding: 3px;

    font-size: 12px;

    padding-left: 5px;

    padding-right: 5px;</string>
      </property>
      <property name="text">
       <string>exportExcel</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../téléchargement.jpg</normaloff>../téléchargement.jpg</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="tri_alpha">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>440</y>
        <width>121</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #b1b1b1;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);

 

    border-width: 3px;

    border-color: #ffffff;

    border-style: solid;

    border-radius: 10;



    padding: 3px;

    font-size: 12px;

    padding-left: 5px;

    padding-right: 5px;
</string>
      </property>
      <property name="text">
       <string>tri alphabetique</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../291613227_370296138513623_6295941384685589861_n.png</normaloff>../291613227_370296138513623_6295941384685589861_n.png</iconset>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="stat_combo_2">
     <attribute name="title">
      <string/>
     </attribute>
     <widget class="QFrame" name="horizontalFrame">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>841</width>
        <height>501</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout"/>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>944</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
